### PR TITLE
PAYARA-3294 Upgrade javax.mail to 1.6.2

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/pom.xml [hunk 1]
```diff
@@ -293,7 +293,7 @@
                                 <version>${mail.version}</version>
                             </artifact>
                             <jarType>impl</jarType>
-                            <specVersion>1.6</specVersion>
+                            <specVersion>${mail.version}</specVersion>
```
Comment (arjantijms) [inline appserver/pom.xml]: I think here too spec version should remain at 1.6, as there's no 1.6.1 spec

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**